MovingBlocks/TerasologyLauncher
# chore: update Gradle (7.1.1 >>> 7.5.1) and dependencies

### Contains

- build: upgrade Gradle (7.1.1 >>> 7.5.1)
- build: upgrade dependencies
- build: enable locked dependencies


### How to test

Build and run the launcher from source via Gradlew and ensure that everything is still working.

To generate or update the lockfiles for both the Java dependencies and the build plugin dependencies, run:

```
gradlew dependencies --write-locks
```

### Outstanding before merging

Nothing.
